R/add_quantiles.R

```diff
@@ -0,0 +1,154 @@
+#' @title Add lines indicating the survival quantiles to a survival plot
+#'
+#' @description Method to add lines indicating the specified quantiles of the survival curve.
```

Comment:
for any plot right?

Reply:
only for survival? then why create a method?

Reply:
Right now we only have survival implemented, but maybe we'll add others? So I stuck with the structure. Also, I copied the file from `add_CI` and then just adjusted from there on. Theoretically we're not yet using that for anything else yet, but who knows.

Reply:
A method should be general so please describe it generally

Reply:
update to keep general.